### PR TITLE
jewel: build/ops: deb: Fix logrotate packaging

### PR DESCRIPTION
**Pull request by smithfarm**
http://tracker.ceph.com/issues/20316

### CODE REVIEW

**Comment by smithfarm** [comment]
Needs to be tested using a technique similar to the one demonstrated by @tchaikov in https://github.com/ceph/ceph/pull/14600#issuecomment-294735825

**Comment by smithfarm** [comment]
https://shaman.ceph.com/builds/ceph/wip-20162-jewel/

**Comment by smithfarm** [comment]
Tested with:

```
$ wget https://3.chacra.ceph.com/r/ceph/wip-20162-jewel/201c6c5b916466ef05246f9cbf01c5eb61a72328/ubuntu/xenial/flavors/default/pool/main/c/ceph/ceph-common_10.2.7-251-g201c6c5-1xenial_amd64.deb
$ wget https://3.chacra.ceph.com/r/ceph/wip-20162-jewel/201c6c5b916466ef05246f9cbf01c5eb61a72328/ubuntu/xenial/flavors/default/pool/main/c/ceph/ceph-base_10.2.7-251-g201c6c5-1xenial_amd64.deb
$ dpkg-deb -x ceph-common_10.2.7-251-g201c6c5-1xenial_amd64.deb c
$ dpkg-deb -x ceph-base_10.2.7-251-g201c6c5-1xenial_amd64.deb d
$ ls -l c/etc/logrotate.d/
ls: cannot access 'c/etc/logrotate.d/': No such file or directory
$ ls -l d/etc/logrotate.d/
total 4
-rw-r--r-- 1 ubuntu ubuntu 228 Jun  2 08:30 ceph-base.logrotate
```

**Comment by smithfarm** [comment]
Marking DNM until we get positive confirmation that this is really necessary. See http://tracker.ceph.com/issues/15569#note-10

**Comment by tchaikov** [comment]
@smithfarm do you think we should backport #15567 as well?

**Comment by smithfarm** [comment]
@tchaikov The jewel debian packaging does not have anything fancy. In #15567 it says: 

>  Backporting to Kraken should work fine, Jewel could do a manual move similar to what is proposed in #15428 , but to ceph-common?

So, if you agree, I'll `s/ceph-base/ceph-common/g` in this patch and call it done?

**Comment by tchaikov** [comment]
@smithfarm i agree, and thanks =)

**Comment by smithfarm** [comment]
@tchaikov Restaged, redone, ready for review.

File: debian/rules [hunk 1]
```diff
@@ -67,7 +67,7 @@ clean:
 	  ltmain.sh missing
 	rm -f configure Makefile.in man/Makefile.in src/Makefile.in
 	rm -f src/acconfig.h.in
-	rm -f debian/ceph-base.ceph.init debian/radosgw.init debian/ceph.logrotate debian/radosgw.logrotate
+	rm -f debian/ceph-base.ceph.init debian/radosgw.init debian/ceph-common.logrotate
```
Comment (smithfarm) [inline debian/rules]: @tchaikov When a current jewel user upgrades to a new version containing this change, is this the line that cleans up the old files? I wonder if we should retain `debian/ceph.logrotate` here.

Comment (tchaikov) [inline debian/rules]: no, these lines are how "make clean" is supposed to cleanup the package maintainer's building directory. it has nothing to do with the end user.

**Pull request merged**